test/libsolidity/syntaxTests/viewPureChecker/access_to_base_member_array.sol

```diff
@@ -3,9 +3,6 @@ contract A {
 }
 
 contract B is A {
-    function f() public view {
-        A.x.length = 2;
```

Comment:
Is there a different test for ``push()`` not to be ``view``?

Reply:
Added.